[2023-06-28T23:36:21.256+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_pnadc_data_new.transfer_s3_to_rds manual__2023-06-28T19:19:18-04:00 [queued]>
[2023-06-28T23:36:21.278+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_pnadc_data_new.transfer_s3_to_rds manual__2023-06-28T19:19:18-04:00 [queued]>
[2023-06-28T23:36:21.279+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-06-28T23:36:21.279+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-06-28T23:36:21.279+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-06-28T23:36:21.399+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): transfer_s3_to_rds> on 2023-06-28 23:19:18+00:00
[2023-06-28T23:36:21.410+0000] {standard_task_runner.py:55} INFO - Started process 11958 to run task
[2023-06-28T23:36:21.416+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_pnadc_data_new', 'transfer_s3_to_rds', 'manual__2023-06-28T19:19:18-04:00', '--job-id', '320', '--raw', '--subdir', 'DAGS_FOLDER/get_pnadc_data_new.py', '--cfg-path', '/tmp/tmpql1_kb0o']
[2023-06-28T23:36:21.418+0000] {standard_task_runner.py:83} INFO - Job 320: Subtask transfer_s3_to_rds
[2023-06-28T23:36:21.570+0000] {task_command.py:389} INFO - Running <TaskInstance: get_pnadc_data_new.transfer_s3_to_rds manual__2023-06-28T19:19:18-04:00 [running]> on host d9b032dfc97f
[2023-06-28T23:36:22.041+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=get_pnadc_data_new
AIRFLOW_CTX_TASK_ID=transfer_s3_to_rds
AIRFLOW_CTX_EXECUTION_DATE=2023-06-28T23:19:18+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-28T19:19:18-04:00
[2023-06-28T23:36:22.054+0000] {base.py:73} INFO - Using connection ID 'aws_conn' for task execution.
[2023-06-28T23:36:22.055+0000] {connection_wrapper.py:337} INFO - AWS Connection (conn_id='aws_conn', conn_type='aws') credentials retrieved from login and password.
[2023-06-28T23:36:27.833+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-06-28T23:41:32.531+0000] {logging_mixin.py:137} INFO - End to add rows! 368210 rows added.
[2023-06-28T23:41:32.541+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-28T23:41:32.557+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=get_pnadc_data_new, task_id=transfer_s3_to_rds, execution_date=20230628T231918, start_date=20230628T233621, end_date=20230628T234132
[2023-06-28T23:41:32.685+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-06-28T23:41:32.840+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
